students/suhun/users/views.py

```diff
@@ -35,4 +35,17 @@ def post(self, request):
                     )
             return JsonResponse({'message':'success'}, status=201)
         except KeyError:
-            return JsonResponse({'message':'keyerror'}, status=401)
\ No newline at end of file
+            return JsonResponse({'message':'keyerror'}, status=401)
+
+
+class SigninView(View):    
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            email    = data["email"]
+            password = data["password"]
+            if not User.objects.filter(email = email, password = password): 
+                return JsonResponse({"message": "INVALID_USER"}, status= 401)
```

Comment:
줄바꿈 해주면 가독성이 더 좋을 것 같네요.
```suggestion
        try:
            data     = json.loads(request.body)
            email    = data["email"]
            password = data["password"]
            
            if not User.objects.filter(email = email, password = password): 
                return JsonResponse({"message": "INVALID_USER"}, status= 401)
```

Reply:
줄바꿈 수정후 커밋했습니다!